react-app/src/components/CodeListing.tsx

```diff
@@ -0,0 +1,47 @@
+import { CSSProperties } from "react";
+import { parseCssEntries } from "../util/util";
+import clipboardSign from "../assets/clipboard-outline-svgrepo-com.svg";
+
+import styles from "../styles/flexbox/CodeListing.module.css";
+export function CodeListing({ cssProps }: { cssProps: CSSProperties }) {
+  function generateEntries() {
+    return parsedEntries.map((e, index) => (
```

Comment:
in JS e is event shortcut, I suggest to use object desructurisation

Reply:
Name fixed to "entry", but I won't change map to destruicturisation. I thing its better there